ResearchKit/ActiveTasks/ORKTowerOfHanoiTowerView.m

```diff
@@ -66,7 +66,8 @@ - (instancetype)initWithFrame:(CGRect)frame maximumNumberOfDisks:(NSUInteger)max
 
 - (void)updateConstraints {
     if (_currentConstraints != nil) {
-        [NSLayoutConstraint deactivateConstraints:_currentConstraints];
+        [self removeConstraints:_currentConstraints];
```

Comment:
I reproduced the crash but I don't see a good reason for it happening. Doesn't it actually look like an *iOS 9 API* bug that should be fixed at the *iOS* level?

This fix goes against my constraint refactor (activate/deactivate is the recommended way of adding/removing constraints [by Apple here](https://developer.apple.com/library/ios/documentation/AppKit/Reference/NSLayoutConstraint_Class/#//apple_ref/occ/instp/NSLayoutConstraint/active)).